students/changmin/users/views.py

```diff
@@ -1,3 +1,41 @@
-from django.shortcuts import render
+import json, re
 
-# Create your views here.
+from django.views           import View
+from django.http            import JsonResponse
+from django.db              import IntegrityError
+from django.core.exceptions import ValidationError
+
+from users.models           import User
+
+class SignUpView(View):
+     def post(self, request):
+        data           = json.loads(request.body)
+        regex_email    = '[a-zA-Z0-9]+\.?\w*@\w+[.]?\w*[.]+\w{2,3}'
+        regex_password = '(?=.*[A-Za-z])(?=.*\d)(?=.*[~!@#$^&*()+|=])[A-Za-z\d~!@#$%^&*()+|=]{8,}'
```

Comment:
regex는 상수이기 때문에 대문자로 선언해주시고, `class SignUpView`의 member 변수로 선언해서 다른 hanlder method에서도 사용할 수 있도록 하면 좋을 것 같습니다.
```suggestion
class SignUpView(View):
    REGEX_EMAIL    = '^[a-zA-Z0-9+-_.]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$'
    REGEX_PASSWORD = '^(?=.*[A-Za-z])(?=.*\d)(?=.*[$@$!%*#?&])[A-Za-z\d$@$!%*#?&]{8,}$'

     def post(self, request):
        data           = json.loads(request.body)
```